Doc/c-api/typeobj.rst

```diff
@@ -1558,6 +1558,8 @@ and :c:type:`PyType_Type` effectively act as defaults.)
    :c:member:`~PyTypeObject.tp_init` function is called; if :c:member:`~PyTypeObject.tp_new` returns an instance of a
    subtype of the original type, the subtype's :c:member:`~PyTypeObject.tp_init` is called.
 
+   Returns ``0`` on success, or return ``-1`` and sets an exception on error.
```

Comment:
I'm not native english, but the asymetry (Return**s** / return) may indicate there is something wrong here.
```suggestion
   Returns ``0`` on success, ``-1`` and sets an exception on error.
```

As a non-english speaker I'd rely on grep but hey grep tels we can use return or returns, I'm lost, but mixing both hurt my eyes.

```
$ git grep -oh 'Returns\? ..0...*,.*on' Doc/*/*.rst | sort | uniq -c | sort -gr
      6 Return ``0`` on success, ``-1`` on
      1 Returns ``0`` on success, and ``-1`` on
      1 Returns ``0`` if successful, a nonzero value on
      1 Return ``0`` on success, return ``-1`` on

```